pytorch_lightning/core/saving.py

```diff
@@ -1,5 +1,8 @@
+import ast
+import collections
```

Comment:
is this import needed? I can't see it.

Reply:
done d922f2f485c10e177ad92ac9d9a64a12085a497b